var/spack/repos/builtin/packages/python/package.py

```diff
@@ -148,6 +148,15 @@ def install(self, spec, prefix):
         make()
         make('install')
 
+        self.sysconfigfilename = '_sysconfigdata.py'
+        if spec.satisfies('@3.6:'):
```

Comment:
I put a comment in there indicating the that config file name changed as of 3.6.0. I'm not sure if you wanted something more detailed, but there's not a whole lot more I can add anyway.